DataRepo/models.py

```diff
@@ -37,6 +37,22 @@ class TracerLabeledClass:
 
     MAX_LABELED_ATOMS = 20
 
+    @classmethod
+    def tracer_labeled_element_regex_pattern(cls):
```

Comment:
NB. This doesn't return a regex pattern.  It returns the string `CHNOS` (or some other order of characters).  I suggest a more accurate name:
```suggestion
    def tracer_labeled_elements_concat_string(cls):
```

Reply:
maybe I will move this method/function to the Loader class, instead.  This method predated the other method I added (`tracer_labeled_elements_list`) , so unless anyone thinks they "will also need a concatenated string", I might go that route.

Reply:
I modified/moved it